src/js/components/search/topFilterBar/filterGroups/DefCodesFilterGroup.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * PSCFilterGroup.jsx
+ * Created by Emily Gullo 07/21/2017
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import BaseTopFilterGroup from './BaseTopFilterGroup';
+
+const propTypes = {
+    filter: PropTypes.object,
+    redux: PropTypes.object,
+    compressed: PropTypes.bool
+};
+
+export default class PSCFilterGroup extends React.Component {
```

Comment:
is this a `DefCodesFilterGroup` or `PSCFilterGroup`?

Reply:
@MaxwellKendall is this not the case? Did the mock change? Please explain why this conversation was resolved?